dot/network/host.go

```diff
@@ -362,6 +362,17 @@ func (h *host) peers() []peer.ID {
 	return h.h.Network().Peers()
 }
 
+// getPeerProtocols returns all the supported protocols to one peer
```

Comment:
`getPeerProtocols` -> `supportsProtocol`

`supportsProtocol` doesn't return all the supported protocols.
`supportsProtocol` checks if `protocol` is supported by `peerID`.

Reply:
Done!